util/image_viewer/spotfinder_frame.py

```diff
@@ -1299,6 +1343,8 @@ def map_coords(x, y, p):
         ctr_mass_dict = {"width": 2, "color": "#FF0000", "closed": False}
         vector_dict = {"width": 4, "color": "#F62817", "closed": False}
         i_frame = self.images.selected.index
+        if self.viewing_stills:
+            i_frame = self.images.selected_index
```

Comment:
At first, I thought this was redundant code, but then I saw the `.`/`_`. Any chance one of the names can be refactored so that they do not look _that_ self-similar?

Reply:
in any case please rewrite as 
```python
if self.viewing_stills:
    i_frame = ...
else:
    i_frame = ...
```
which will align the expressions and make the difference more obvious

Reply:
Both of those names are seemingly difficult to change, one is the ImageCollectionWithSelection (self.images) attribute, and the other is an attribute of the actual objects being store in the OrderedSet (parent of ImageCollectionWithSelection) .. I agree it's a bit confusing to read  ..